_posts/2020-06-25-changelog-125.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #125"
+date: 2020-06-25 19:26:11 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5535 -->
+* Support debugging with MSVC toolchain in CLion 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5586 -->
+* Add `Introduce local variable` intention (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5626 -->
+* Implement `Move File` refactoring
+
+    TODO: GIF?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5614 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5601 -->
+* Add intentions to convert a method call to UFCS and vice versa (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5550 -->
+* Add the quick fix to convert an immutable reference to a mutable one (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5557 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5561 -->
+* Add quick fixes to remove unused variable / parameter (by [@Kobzol])
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5609 -->
+* Update icons for Rust items
+
+    TODO: PNG (old / new)?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/4640 -->
+* Enable intelligent highlighting inside macro call bodies
+
+* Add detection of the new compiler errors:
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5375 -->
+  * You tried to use a type which doesn't implement some trait in a place which expected that trait [E0277](https://doc.rust-lang.org/error-index.html#E0277) (by [@Kobzol])
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5436 -->
+  * A constant item was initialized with something that is not a constant expression [E0015](https://doc.rust-lang.org/error-index.html#E0015) (by [@t-kameyama])
+  * A non-constant value was used in a constant expression [E0435](https://doc.rust-lang.org/error-index.html#E0435) (by [@t-kameyama])
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5582 -->
+  * A representation attribute was placed on an unsupported item [E0517](https://doc.rust-lang.org/error-index.html#E0517) (by [@Stzx])
+  * An unrecognized representation attribute was used [E0552](https://doc.rust-lang.org/error-index.html#E0552) (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5571 -->
+* Now IDE's inspections take into account lint level attributes (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5466 -->
+* Support for `non_exhaustive` attribute (by [@Mcat12])
```

Comment:
But it's not true - `non_exhaustive ` attribute had been supported in `Match check` inspection before this fix.
The corresponding PR fixes false positive "Match must be exhaustive" error